lntest/node.go

```diff
@@ -1590,3 +1535,81 @@ func (hn *HarnessNode) WaitForBalance(expectedBalance btcutil.Amount, confirmed
 
 	return nil
 }
+
+// makeOutpoint returns the outpoint of the channel's funding transaction.
+func makeOutpoint(chanPoint *lnrpc.ChannelPoint) (wire.OutPoint, error) {
+	fundingTxID, err := lnrpc.GetChanPointFundingTxid(chanPoint)
+	if err != nil {
+		return wire.OutPoint{}, err
+	}
+	return wire.OutPoint{
+		Hash:  *fundingTxID,
+		Index: chanPoint.OutputIndex,
+	}, nil
+}
+
+// handleChannelEdgeUpdates takes a series of channel edge updates, extracts
+// the outpoints, and saves them to harness node's internal state.
+func (hn *HarnessNode) handleChannelEdgeUpdates(
+	updates []*lnrpc.ChannelEdgeUpdate) {
+
+	// For each new channel, we'll increment the number of
+	// edges seen by one.
+	for _, newChan := range updates {
+		op, err := makeOutpoint(newChan.ChanPoint)
+		if err != nil {
+			hn.AddToLog("failed to create outpoint for %v"+
```

Comment:
Do we need to return here to make sure we don't use an invalid outpoint? Or pass in `t` to fail the test?

Reply:
Added a `return` here.
We could perhaps use `panic` instead?

Reply:
So I ended up printing the error to the console. Thought about using `panic` which made me panic,  being afraid of it would panic where it shouldn't panic😂
Also the nits are fixed!